control_panel_api/aws.py

```diff
@@ -50,6 +51,26 @@ def create_policy(self, name, policy_document):
             PolicyName=name,
             PolicyDocument=json.dumps(policy_document))
 
+    def get_inline_policy_document(self, role_name, policy_name):
+        try:
+            result = self._do('iam', 'get_role_policy',
+                RoleName=role_name,
+                PolicyName=policy_name,
+            )
+            return result['PolicyDocument']
+        except ClientError as e:
```

Comment:
I just noticed your error handling here.  This kind of duplicates the work done in services.py:21 where a decorator is used to ignore aws error.  Have a look see if that is better suited.

Reply:
Cheers, I'll have a look 👍 - I _may_ slip this change in the clean-up PR - I'm kind of keen to merge ASAP tho so I may tweak this independently to avoid conflicts, etc...